File: pattern2.c
#include <stdio.h>
int main()
{
    int n;
    printf("Enter no of lines:");
    scanf("%d",&n);
    for(int i=n;i>=1;i--)
    {
        for(int j=0;j<=i-1;j++)
        {
            printf(" ");
        }
        for(int k=n;k>=i;k--)
        {
            printf("%d",k);
        }
        for(int l=i+1;l<=n;l++)
        {
            printf("%d",l);
        }
        printf("\n");
    }
    for(int i=n;i>=1;i--)
    {
        for(int j=0;j<=n-i+1;j++)
        {
            printf(" ");
        }
        for(int k=n;k>=n-i+2;k--)
        {
            printf("%d",k);
        }
        for(int l=n-i+3;l<=n;l++)
        {
            printf("%d",l);
        }
        printf("\n");
    }
    
    return 0;
}

File: Assignment_1/pattern2.py
n=int(input("Enter no of lines:"))
for i in range(n,0,-1):
    for j in range(1,i):
        print(' ',end='')
    for k in range(n,i-1,-1):
        print(kend='')
    for l in range(i+1,n+1):
        print(l,end='')
    print()

for i in range(n,0,-1):
    for j in range(0,n-i+2):
        print(' ')
    for k in range(n,n-i+3,-1):
        print(k)
    for l in range(n-i+3,n+1):
        print(l)
    print()